### PR TITLE
Update templates to use TT for uppercase text

### LINKED ISSUE #7658: Update templates to use new uppercase function
## Feature Description

Once we have a toggle to make text uppercase, we should update all templates **plus** the Get Started story to make use of it. No longer ALL CAPS text, yay!

This also applies to text sets.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Context

With the addition of uppercase toggle for text, template JSON needed to be updated to use TT.

## Summary

Updates templates to use `text-transform` to make text uppercase, for following templates:

- Fresh & Bright
- Food & Stuff
- Doers Get More Done
- Weekly Entertainment
- Fashion On The Go
- No Days Off
- Experience Thailand
- Sleep
- Baking Bread Guide
- Trendy Winter Veggie
- Ways to Eat Avocado
- Kitchen Stories
- Album Releases

## Testing Instructions

- Go to Explore Templates screen, path `wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery`
- Pick any one from the above listed templates and create a new story by clicking `Use Template` button.
- Select an uppercase word, observe that the `TT` button is highlighted in the settings panel.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7658

### CODE REVIEW

**Comment by divyarajmasani** [comment]
This PR only addresses the templates for now, do we want to update text sets in the same PR or it is to be taken care in a separate PR? @swissspidy

**Comment by swissspidy** [comment]
> This PR only addresses the templates for now, do we want to update text sets in the same PR or it is to be taken care in a separate PR? @swissspidy

I think we can do text sets separately, but would be able to update our demo story as well?

It works a bit different from the templates though, so requires some changes in the code as well.

The story content is here:

https://github.com/google/web-stories-wp/blob/9c956c2546c26137631ccb40b90f0bc08bf00e32/includes/data/stories/demo.json

But the text elements just contain placeholders, which are then replaced here:

https://github.com/google/web-stories-wp/blob/9c956c2546c26137631ccb40b90f0bc08bf00e32/includes/Demo_Content.php#L86-L158

So basically need to:

1. Add `text-transform: uppercase` in `demo.json` where applicable
2. Update strings in `Demo_Content`

Edit: the demo story can be viewed via `/wp-admin/post-new.php?post_type=web-story&web-stories-demo=1`

**Comment by swissspidy** [comment]
> > This PR only addresses the templates for now, do we want to update text sets in the same PR or it is to be taken care in a separate PR? @swissspidy
> 
> I think we can do text sets separately, but would be able to update our demo story as well?

Or well, how much effort would it be to update those too in this PR?

**Comment by divyarajmasani** [comment]
> Or well, how much effort would it be to update those too in this PR?

Not sure how many text sets need to be updated but I just glanced over there are some 70 text sets so I believe it could take around 2-3 hrs.

**Comment by divyarajmasani** [comment]
> > This PR only addresses the templates for now, do we want to update text sets in the same PR or it is to be taken care in a separate PR? @swissspidy
> 
> I think we can do text sets separately, but would be able to update our demo story as well?
> 
> It works a bit different from the templates though, so requires some changes in the code as well.
> 
> The story content is here:
> 
> https://github.com/google/web-stories-wp/blob/9c956c2546c26137631ccb40b90f0bc08bf00e32/includes/data/stories/demo.json
> 
> But the text elements just contain placeholders, which are then replaced here:
> 
> https://github.com/google/web-stories-wp/blob/9c956c2546c26137631ccb40b90f0bc08bf00e32/includes/Demo_Content.php#L86-L158
> 
> So basically need to:
> 
> 1. Add `text-transform: uppercase` in `demo.json` where applicable
> 2. Update strings in `Demo_Content`
> 
> Edit: the demo story can be viewed via `/wp-admin/post-new.php?post_type=web-story&web-stories-demo=1`

sure, I'll go through this and update the PR.

**Comment by divyarajmasani** [comment]
I've updated the demo story JSON.

**Comment by swissspidy** [comment]
Awesome, thanks! Change looks good there too.

I think this is ready to 🚢 

Given the amount of work needed for updating the text sets, let's do those in a separate PR.

**Review by spacedmonkey — APPROVED** [review]
